reference/configuration/framework.rst

```diff
@@ -55,7 +55,7 @@ Configuration
 * `session`_
     * `storage_id`_
     * `handler_id`_
-    * `name`_
+    * :ref:`name <session-name>`
```

Comment:
@xabbuh adding the `.. _name:` label manually before the existing `name` option wouldn't work?